Tests/FluentAssertions.Shared.Specs/XDocumentAssertionSpecs.cs

```diff
@@ -17,7 +17,7 @@ public class XDocumentAssertionSpecs
         #region Be / NotBe
 
         [TestMethod]
-        public void When_asserting_a_xml_document_is_equal_to_the_same_xml_document_it_should_succeed()
+        public void When_asserting_a_XDocument_is_equal_to_the_same_XDocument_it_should_succeed()
```

Comment:
I assume these are the renames you're thinking of. I read the principles and I think that in this case it still makes sense to name the test after the class. The principle (which is sound) is to not tie test names to *internal* names of the system, which in this case is FA. But here I'm tying names to the *external* interface being tested - i.e. the .NET API names (which will not be changed).

But that's just my 2c, so if you want I can spend 2 minutes reverting the renames.